spec/spec_helper.rb

```diff
@@ -4,10 +4,32 @@
 # file to always be loaded, without a need to explicitly require it in any files.
 
 require_relative '../twitter.rb'
+require_relative './support/vcr_setup.rb'
 
 require 'base64'
 
+require 'webmock/rspec'
+WebMock.disable_net_connect!(allow_localhost: true)
+
+VCR.configure do |c|
+  #the directory where your cassettes will be saved
+  c.cassette_library_dir = 'spec/vcr'
+  # your HTTP request service. You can also use fakeweb, webmock, and more
+  c.hook_into :webmock
+end
+
 RSpec.configure do |config|
   config.tty = true
   config.order = 'default'
 end
+
+# {client: {
+#    access_token: "2911734821-Grm2qtLudgA8C9G2N5NeqNP74Q8Cdg4zgzGr02H",
```

Comment:
oops. no, those are not my real keys. those were an effort at stubbing. delete comment though.